src/AutoSysimages.jl

```diff
@@ -444,6 +442,55 @@ If this not works please add the script into your `PATH`."""
 
 end
 
+"""
+    uninstall(;verbose::Bool = true, dry::Bool = false)
+
+This removes the `asysimg` scripts and all the generated files (by calling `cleanup`).
+"""
+function uninstall(dir = _default_install_dir();verbose::Bool = true, dry::Bool = false)
+    file_name = Sys.iswindows() ? "asysimg.bat" : "asysimg"
```

Comment:
Check if the default or specified install dir exists

```julia
if isnothing(dir)
        @warn """Install directory not found."""
        return
end
```

Reply:
Fixed by e0521507826e4ba60b12ec566746aa6999749237, thanks.